src/js/components/molecules/list-unordered-limit/list-unordered-limit.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, Mixins, Prop } from 'vue-property-decorator';
+import { default as Events } from 'constants/events';
+import { ListUnorderedLimitProps } from './list-unordered-limit.models';
+import { StepRowAnimation } from 'js/mixins/step-row-animation/step-row-animation';
+import { default as ViewportUtil } from 'utils/viewport';
+import WithRender from './list-unordered-limit.vue.html';
+
+@WithRender
+@Component({
+    name: 'ListUnorderedLimit',
+})
+
+export class ListUnorderedLimit extends Mixins(StepRowAnimation) implements ListUnorderedLimitProps {
+    @Prop({ type: Object, required: true }) listItems!: { items: [string[] | object[]], type: string };
```

Comment:
We should not be creating our own types.

Reply:
resolved in discussion with @zwaardje